docs/plugins/duplicates.rst

```diff
@@ -46,8 +46,8 @@ file. The available options mirror the command-line options:
 - **checksum**: Use an arbitrary command to compute a checksum
   of items. This overrides the ``keys`` option the first time it is run;
   however, because it caches the resulting checksum as ``flexattrs`` in the
-  database, you can use ``--keys=name_of_the_checksumming_program
-  any_other_keys`` (or set configuration ``keys`` option) the second time
+  database, you can use ``--key=name_of_the_checksumming_program
+  --key=any_other_keys`` (or set configuration ``keys`` option) the second time
```

Comment:
This isn't your fault, but the parenthetical should probably read "or set the keys configuration option".